stable/coredns/templates/rbac.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if (.Values.middleware.kubernetes.enabled) and (.Values.rbac.create) }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "coredns.fullname" . }}
```

Comment:
Please add labels `chart`, `heritage`, `release`, `app` to all RBAC resources. You can copy them from the deployment.